src/js/setUpSite.ts

```diff
@@ -161,16 +161,16 @@ const snapToCity: (map: Map, layer: ImageOverlay) => void = async (
 };
 
 /**
- * Sets scorecard to city.
+ * Sets scorecard to city and alert if city is community made.
  *
  * @param cityId: E.g. `columbus-oh`.
  * @param cityProperties: An object with a `layout` key (Leaflet value) and keys
  *    representing the score card properties stored in `score-cards.json`.
  */
-const setScorecard: (cityId: CityId, cityProperties: ScoreCard) => void = (
-  cityId,
-  cityProperties
-) => {
+const setScorecardAndAlert: (
+  cityId: CityId,
+  cityProperties: ScoreCard
+) => void = (cityId, cityProperties) => {
```

Comment:
this might mess up formatting

```suggestion
const setScorecardAndAlert = (cityId: CityId, cityProperties: ScoreCard): void => {
```